### ISSUE TITLE
Electron

### ISSUE THREAD

**Issue by uwenayoallain** [microsoft/vscode#116193]
Issue Type: <b>Bug</b>

(electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.

VS Code version: Code 1.53.0 (8490d3dde47c57ba65ec40dd192d014fd2113496, 2021-02-03T20:36:38.611Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6560U CPU @ 2.20GHz (4 x 2208)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|6.16GB (1.42GB free)|
|Process Argv|--crash-reporter-id e88e323e-0608-496e-a85f-98e9df1bb54e|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (29)</summary>

Extension|Author (truncated)|Version
---|---|---
project-manager|ale|12.0.1
vscode-intelephense-client|bme|1.6.3
vscode-tailwindcss|bra|0.5.9
gitignore|cod|0.6.0
bracket-pair-colorizer|Coe|1.0.61
vscode-eslint|dba|2.1.14
git-extension-pack|don|0.1.3
githistory|don|0.6.14
es7-react-js-snippets|dsz|3.1.0
gitlens|eam|11.2.1
vscode-html-css|ecm|1.9.1
prettier-vscode|esb|5.9.1
php-debug|fel|1.14.9
php-intellisense|fel|2.3.14
php-pack|fel|1.0.2
auto-complete-tag|for|0.1.0
auto-rename-tag|for|0.1.6
code-runner|for|0.11.2
beautify|Hoo|1.5.0
vs-color-picker|lih|1.0.0
dotenv|mik|1.0.1
mongodb-vscode|mon|0.4.0
open-html-in-browser|pea|2.1.3
px-to-rem|sai|1.3.1
win-opacity|ska|1.1.4
tabnine-vscode|Tab|3.2.6
vscode-icons|vsc|11.1.0
file-icons-mac|way|1.0.1
vscode-open-in-github|ziy|1.3.6

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
pythonvspyt617cf:30253240
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30211402
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt600cf:30251589

```

</details>

**Comment by uwenayoallain** [comment]
Where is this error coming from?

**Comment by deepak1556** [comment]
Its from the crash-reporter in electron, there is no timeline for when this will be fixed on our end.

**Comment by iota-008** [comment]
it comes when i run npm start

`
server :: master ❯ npm start

> server@1.0.0 start
> index.js


(electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.

`

**Comment by ehabwalied262** [comment]
I am using nodemon and this issue is annoying me , when i run without nodemon it works fine !!

**Comment by susananzth** [comment]
I had the same error, the problem in my case is the name of the archive. I renamed the archive and problem resolved :S

**Comment by jasondhakhwa** [comment]
> I had the same error, the problem in my case is the name of the archive. I renamed the archive and problem resolved :S

Please teach the steps

**Comment by GitMensch** [comment]
I've just reproduced the electron warning with running

~~~
code --diff somefile path-to-not-exisiting-file

electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.
~~~

@deepak1556 wrote:
> Its from the crash-reporter in electron, there is no timeline for when this will be fixed on our end.

So it seems to be related to the electron update, isn't it? I'm completely fine with "no timeline", but then there likely is an open issue on the backlog to solve this *somedayTM*, isn't it? [If not - I did not found it on the first page with a search - then it would be good to have one, even if it just contains the quote above which says "we know about this and acknowledge it as an issue so it is kept open and on the backlog"]

**Comment by deepak1556** [comment]
Please use this to workaround,

```
* Open the Command Palette.
* Run the Preferences: Configure Runtime Arguments command.
This command will open a argv.json file to configure runtime arguments. You might see some default arguments there already.
* "enable-crash-reporter":  false
* Restart VS Code.
```

**Comment by vivekkupadhyay** [comment]
> To clarify this is not a bug, just a warning to adapt upcoming changes in crash reporter of electron which is being tracked internally, if the log is interrupting any workflow please use this to workaround
> 
> ```
> * Open the Command Palette.
> * Run the Preferences: Configure Runtime Arguments command.
> This command will open a argv.json file to configure runtime arguments. You might see some default arguments there already.
> * "enable-crash-reporter":  false
> * Restart VS Code.
> ```

Thanks for the update, but it still not working for me.

**Comment by jasondhakhwa** [comment]
im getting the same problem in django idk why, in every django project I get the same problem even in new empty django project

![image](https://user-images.githubusercontent.com/71743724/109009836-240fe600-76d7-11eb-9579-185d297b0ef7.png)

**Comment by jasondhakhwa** [comment]
how to run Preferences: Configure Runtime Arguments command?

**Comment by deepak1556** [comment]
This is already fixed in our insiders https://code.visualstudio.com/insiders and will be fixed in the next stable release `1.54`

**Comment by jasondhakhwa** [comment]
im getting this problem in cmd as well its not only vs code

**Comment by Karan-Dhingra** [comment]
(electron) Sending uncompressed crash reports is deprecated and will be removed in a future version of Electron. Set { compress: true } to opt-in to the 
new behavior. Crash reports will be uploaded gzipped, which most crash reporting servers support.

I am getting this in the terminal when I run CPP Program Please help me to resolve this issue.

**Comment by ClutchDown** [comment]
> This is already fixed in our insiders https://code.visualstudio.com/insiders and will be fixed in the next stable release `1.54`

I can try it, but it doesnt work :(
